### DISCUSSION TITLE
allways thougt that also free acconts were able to run  wikis  pages

### DISCUSSION THREAD

**Discussion by fsj-digital** [community/community#53424] [New to GitHub]
did i get it wrong 

allways thougt that also free acconts were able to run 
wikis 
pages 

but if we look at the pricing list - there other things are noted. 

this is bit confusing

![image](https://user-images.githubusercontent.com/32641183/233674413-9e45b627-5edb-4434-b23e-51279766ec57.png)

**Comment by NJacksonDev** [comment]
Yes, unless something has very recently changed, free GitHub accounts can create and run wikis (with some limitations on the free account, like a 1GB storage cap). 

- To create a wiki on GitHub, you first need to create a repository for your project. Once you have created the repository, you can then enable the wiki feature by going to the repository's settings page and checking the "Wikis" checkbox under the Features section.

- Once the wiki feature is enabled, you can create and edit wiki pages using GitHub's markdown editor. You can also collaborate with others on the wiki by giving them write access to the repository.
